### PR TITLE
Missing Implementation of `AngleAxisTemplate(const Vector3& angleAxis)`

### PR DESCRIPTION
**Pull request by othlu**

### CODE REVIEW

File: minkindr/include/kindr/minimal/implementation/angle-axis-inl.h [hunk 1]
```diff
@@ -62,6 +62,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
   // \todo furgalep Check that the matrix was good...
 }
 
+template<typename Scalar>
+AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
+    Scalar norm = angleAxis.norm();
+    CHECK_GT(norm, 1e-8);
+    Vector3 axis = angleAxis;
+    axis.normalize();
```
Comment (schneith) [inline minkindr/include/kindr/minimal/implementation/angle-axis-inl.h]: reuse the result of norm() to speed up the normalization

**Comment by schneith** [comment]
add to whitelist

**Comment by schneith** [comment]
lgtm

**Comment by helenol** [comment]
@othlu could you merge this please? :)

**Comment by othlu** [comment]
Would love to but I don't seem to have write access in here...

**Pull request merged**